copier/main.py

```diff
@@ -558,6 +558,13 @@ def _render_path(self, relpath: Path) -> Optional[Path]:
             if not part:
                 return None
             rendered_parts.append(part)
+
+        # The following logic was added based on the discussion: https://github.com/copier-org/copier/discussions/859
+        if len(rendered_parts) > 1 and str(self.answers_relpath).startswith(
+            rendered_parts[0]
+        ):
+            rendered_parts.pop(0)
```

Comment:
This also doesn't seem like the fix you're looking for. If there were more files under `.config/`, they'd be affected too, right?

Reply:
Good point, I will see if I can make it more specific tailored to the answersfile!

Reply:
I have changed the implementation and added more tests, so different config folder levels are tested as well